unit-tests/dds/test-device-init.py

```diff
@@ -0,0 +1,104 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2022 Intel Corporation. All Rights Reserved.
+
+#test:donotrun:!dds
+
+import pyrealdds as dds
+from rspy import log, test
+
+
+dds.debug( True, 'C  ' )
+log.nested = 'C  '
+
+
+participant = dds.participant()
+participant.init( 123, "test-device-init" )
+
+info = dds.device_info()
+info.name = "Test Device"
+info.topic_root = "realdds/device/topic-root"
+
+
+import os.path
+cwd = os.path.dirname(os.path.realpath(__file__))
+remote_script = os.path.join( cwd, 'device-init-server.py' )
+second_client_script = os.path.join( cwd, 'device-2nd-client.py' )
+with test.remote( remote_script, nested_indent="  S" ) as remote:
+    remote.wait_until_ready()
+    #
+    #############################################################################################
+    #
+    test.start( "Test 1 stream..." )
+    try:
+        remote.run( 'test_one_stream()', timeout=5 )
+        device = dds.device( participant, participant.create_guid(), info )
+        device.run()  # If no device is available in 30 seconds, this will throw
+        test.check( device.is_running() )
+        test.check_equal( device.n_streams(), 1 )
```

Comment:
Maybe a different test case, but need to check also that the profile details were received correctly

Reply:
Yeah, this was written before your own PR with the streams was in.
Will be added.